Dockerfile

```diff
@@ -663,6 +665,8 @@ COPY --from=pkg-grub / /
 COPY --from=unicode-pf2 /usr/share/grub/unicode.pf2 /usr/share/grub/unicode.pf2
 
 FROM alpine:3.17.2 AS installer-image
+ARG SOURCE_DATE_EPOCH
```

Comment:
`SOURCE_DATE_EPOCH` comes from buildkit?

Reply:
we inject it